commit 8be6749a7e6201bcbd462ffb25b17ac0e73df95e
Author: Michael McCandless <mikemccand@apache.org>
Date:   Sun Nov 1 16:40:05 2009 +0000

    add CHANGES entry for oal.Lock.isLocked now throwing ioe
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/java/trunk@831699 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/CHANGES.txt b/CHANGES.txt
index 4eb97e0..1bbc1d2 100644
--- a/CHANGES.txt
+++ b/CHANGES.txt
@@ -10,6 +10,8 @@ Changes in backwards compatibility policy
   needs to be recompiled against Lucene 3.0 in order to work. The previously
   deprecated IndexCommitPoint is also removed. (Michael Busch)
 
+* oal.Lock.isLocked is now allowed to throw an IOException
+
 Changes in runtime behavior
 
 * LUCENE-1677: Remove the system property to set SegmentReader class
